scripts/ci/upload_artifacts.sh

```diff
@@ -9,21 +9,11 @@ then
 
   if [[ "${CI_TAG}" ]];
   then
-    if [[ ${ALL_FILES_ACCESS} == "ON" ]]; then
-      #FILENAME_AAB="qfield-${CI_TAG}-${TRIPLET}.aab"
-      FILENAME_APK="qfield-${CI_TAG}-${TRIPLET}.apk"
-    else
-      #FILENAME_AAB="qfield-${CI_TAG}-${TRIPLET}.aab"
-      FILENAME_APK="qfield-${CI_TAG}-${TRIPLET}.apk"
-    fi
+    #FILENAME_AAB="${APP_PACKAGE_NAME}-${CI_TAG}-${TRIPLET}.aab"
+    FILENAME_APK="${APP_PACKAGE_NAME}-${CI_TAG}-${TRIPLET}.apk"
   else
-    if [[ ${ALL_FILES_ACCESS} == "ON" ]]; then
-      #FILENAME_AAB="qfield-plus-dev-${CI_UPLOAD_ARTIFACT_ID}-${CI_COMMIT}-${TRIPLET}.aab"
-      FILENAME_APK="qfield-plus-dev-${CI_UPLOAD_ARTIFACT_ID}-${CI_COMMIT}-${TRIPLET}.apk"
-    else
-      #FILENAME_AAB="qfield-dev-${CI_UPLOAD_ARTIFACT_ID}-${CI_COMMIT}-${TRIPLET}.aab"
-      FILENAME_APK="qfield-dev-${CI_UPLOAD_ARTIFACT_ID}-${CI_COMMIT}-${TRIPLET}.apk"
-    fi
+    #FILENAME_AAB="${APP_PACKAGE_NAME}-dev-${CI_UPLOAD_ARTIFACT_ID}-${CI_COMMIT}-${TRIPLET}.aab"
+    FILENAME_APK="${APP_PACKAGE_NAME}-dev-${CI_UPLOAD_ARTIFACT_ID}-${CI_COMMIT}-${TRIPLET}.apk"
```

Comment:
actually you can remove line 10 to 17 by this
```suggestion
    FILENAME_APK="${APP_PACKAGE_NAME}-${CI_PACKAGE_FILE_SUFFIX}-${TRIPLET}.apk"
```

Reply:
it's all defined in env_gh

Reply:
@3nids , yeah, true.